utils/base64.js

```diff
@@ -0,0 +1,21 @@
+exports.decode_image = (data_string) => {
```

Comment:
I used image-to-base64 library. How is this better?

Reply:
It's just written in the house and there are a few particular specifications!